cluster/gce/gci/configure-helper.sh

```diff
@@ -622,6 +622,7 @@ function write-pki-data {
   if [[ -n "${KUBE_PKI_READERS_GROUP:-}" ]]; then
     (umask 027; echo "${data}" | base64 --decode > "${path}")
     chgrp "${KUBE_PKI_READERS_GROUP:-}" "${path}"
+    chmod g+r "${path}"
```

Comment:
When will these files be created already? Do we know their permissions are always 700? Does this issue happen to `umask 077` below?

Reply:
In gke if you upgrade the master these files already exist. It happens if you upgrade from a version that did not have PR https://github.com/kubernetes/kubernetes/pull/95418 to a version where you had it. This bug will not affect components that run as root only components that run as non-root on the gke-control-plane and need to access these secrets which is exactly one component. 
When the code with umask 077 gets run no component is running as non root (all are running as root) so they are unaffected.

Reply:
If the master is downgraded from umask 027 to 077, are we concerned about leaking read access to certain group?

Reply:
To join a group you need root access, so to join that group an attacker would need root access at which point they already have access to everything. So I don't think it is a concern.

Reply:
SGTM. Thanks